lib/mariaex/connection.ex

```diff
@@ -208,7 +208,9 @@ defmodule Mariaex.Connection do
   end
 
   defp command({:query, statement, params, _opts}, s) do
-    Protocol.send_query(statement, params, s)
+    statement
```

Comment:
Best place for fixing it, would be:

  defp get_command(statement) when is_binary(statement) do
    statement |> String.split(" ", parts: 2) |> hd |> String.downcase |> String.to_atom
  end

Replace `String.split` to `:binary.split([" ", "\n"])`